### DISCUSSION TITLE
Where to download the source code of the latest version WSA-Linux-Kernel?

### DISCUSSION THREAD

**Discussion by dczuo** [microsoft/WSA#68] [General]
[3rdpartysource](https://3rdpartysource.microsoft.com) is only available up to version Public Preview (July [2022).
Are there plans to create repository WSA-Linux-Kernel (similar to [WSL2-Linux-Kernel](https://github.com/microsoft/WSL2-Linux-Kernel)) to provide source code？

**Comment by Mids9** [comment]
https://github.com/microsoft/WSA/discussions/68#discussion-4491815
